cordformation/src/test/resources/postgres.gradle

```diff
@@ -0,0 +1,120 @@
+ext {
+    postgresql_version     = '42.2.12'
+    postgres_image_version = '12'
+    dbUser                 = 'myuser'
+    dbPassword             = 'mypassword'
+    dbSchema               = 'myschema'
+    dbName                 = 'mydb'
+    dbPort                 = 5432
+    dbHostName             = 'localhost'
+    dbInit                 = 'Postgres_init.sh'
+    dbDockerfile           = 'Postgres_Dockerfile'
+    postgres = [
+            dataSourceProperties: [
+                    dataSourceClassName: 'org.postgresql.ds.PGSimpleDataSource',
+                    dataSource: [
+                            user    : dbUser,
+                            password: dbPassword,
+                            url     : "jdbc\":\"postgresql\":\"\"//\"\${DBHOSTNAME}\":\"\${DBPORT}/\${DBNAME}\"?\"currentSchema\"=\"\${DBSCHEMA}",
+                            urlArgs : [
+                                    DBHOSTNAME  : dbHostName,
+                                    DBPORT      : dbPort,
+                                    DBNAME      : dbName,
+                                    DBSCHEMA    : dbSchema
+                            ]
+                    ]
+            ],
+            database: [
+                    transactionIsolationLevel: 'READ_COMMITTED',
+                    runMigration             : true,
+                    schema                   : dbSchema
+            ],
+            dockerConfig: [
+                    dbDockerfile    : dbDockerfile,
+                    dbDockerfileArgs: [
+                         DBNAME         : dbName,
+                         DBSCHEMA       : dbSchema,
+                         DBUSER         : dbUser,
+                         DBPASSWORD     : dbPassword,
+                         DBPORT         : dbPort
+                    ],
+                    dbUser          : dbUser,
+                    dbPassword      : dbPassword,
+                    dbSchema        : dbSchema,
+                    dbName          : dbName,
+                    dbPort          : dbPort,
+                    dbHostName      : dbHostName,
+                    dbDatabase      : dbName
+					]
```

Comment:
Am I _completely_ pedantic about whitespace? Err, "yes" actually :wink:. This `]` bracket is _waaaaaaay_ out...